semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/Settings.scala

```diff
@@ -1,31 +1,56 @@
 package scala.meta.internal.metacp
 
+import java.io.PrintStream
+
+import org.langmeta.io.AbsolutePath
+import org.langmeta.io.Classpath
+
+import scala.meta.cli.Metacp
+
 final case class Settings(
```

Comment:
This can no longer be an internal class if it's used in the public API.

Reply:
Good catch! Will fix.

Reply:
https://github.com/scalameta/scalameta/pull/1394